tianshou/utils/logger/wandb.py

```diff
@@ -36,9 +39,79 @@ def __init__(
         train_interval: int = 1000,
         test_interval: int = 1,
         update_interval: int = 1000,
+        save_interval: int = 1000,
+        project: str = 'tianshou',
+        name: Optional[str] = None,
+        run_id: Optional[str] = None
     ) -> None:
         super().__init__(train_interval, test_interval, update_interval)
+        self.last_save_step = -1
+        self.save_interval = save_interval
+        self.restored = False
+
+        self.wandb_run = wandb.init(
```

Comment:
Could you also please add `entity` parameter to this logger?
Used like this:
```python
wandb.init(project='blah', entity='my-team')
```

Reply:
That's a great catch. I'll add support for entity in this PR